### PR TITLE
Use HTTPS for Oracle repositories

### PR DESCRIPTION
**Pull request by juliogonzalez**
## What does this PR change?

Use HTTPS for Oracle repositories

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: A mention on the release notes will be needed.

- [x] **DONE**

## Test coverage
- No tests: Tool not covered by testing right now.

- [x] **DONE**

## Links

None

- [x] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_lint_checkstyle"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "ruby_rubocop"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

File: utils/spacewalk-common-channels.ini [hunk 1]
```diff
@@ -1199,95 +1199,95 @@ repo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/
...
 
 [oraclelinux6-spacewalk24-client]
 label    = %(base_channel)s-spacewalk24-client
 archs    = %(_x86_archs)s
 name     = Spacewalk 2.4 Client for Oracle Linux 6 (%(arch)s)
 base_channels = oraclelinux6-%(arch)s
-repo_url = http://yum.oracle.com/repo/OracleLinux/OL6/spacewalk24/client/%(arch)s/
+repo_url = https://yum.oracle.com/repo/OracleLinux/OL6/spacewalk24/client/%(arch)s/
 
 [oraclelinux6-scl]
```
Comment (mcalmer) [inline utils/spacewalk-common-channels.ini:1292]: Can also be changed to https.

Comment (juliogonzalez) [reply utils/spacewalk-common-channels.ini:1292]: What do you mean :-?

I see the diff and it's now `https`.

Comment (juliogonzalez) [reply utils/spacewalk-common-channels.ini:1292]: Ah, saw it now. You mean `oraclelinux6-scl` itself. Sorry.

Comment (mcalmer) [reply utils/spacewalk-common-channels.ini:1292]: Yes, I could not add the comment at that place

**Pull request merged**